src/common/types/bookingTypes.ts

```diff
@@ -5,4 +5,15 @@ export type bookingOrderType = {
     time: string,
     amountofpeople: number
 }
+export type bookingTablesType = {
```

Comment:
Типы лучше везде называть с большой буквы
```suggestion
export type BookingTablesType = {
```